addons/mail/models/ir_actions_server.py

```diff
@@ -27,7 +27,26 @@ class IrActionsServer(models.Model):
         }
     )
     # Followers
+    followers_type = fields.Selection(
+        selection=[
+            ('specific', 'Specific Followers'),
+            ('generic', 'Dynamic Followers'),
+        ],
+        help="""
+            - Specific Followers: select specific contacts to add/remove from record's followers.
+            - Dynamic Followers: all contacts of the chosen record's field will be added/removed from followers.
+        """,
+        string='Followers Type',
+        compute='_compute_followers_type',
+        readonly=False, store=True
+    )
+    followers_partner_field_name = fields.Char(
```

Comment:
This should clearly be protected (using a constraint probably), as you do not check what the field is ... field should be a m2o or a 2many linking res.partner comodel. Suddenly if you link, let us say, users because you don't really know how to distinguish them, cracboum ! Better safe than sorry.

Reply:
Thanks ! When we are pinged we try to have a look under a reasonable time :)